docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -708,6 +708,16 @@ WITH (
 )
 AS SELECT name, comment, regionkey FROM tpch.tiny.nation;
 ```
+(delta-lake-shallow-clone)=
+#### Shallow clone
+
+The connector supports reading from shallow cloned tables. A shallow clone is a
+clone that does not copy the data files to the clone target, only the metadata.
+Any changes made to shallow clones affect only the clones themselves and not the
```

Comment:
This is irrelevant since we only support reading in Trino. Remove the sentence.

```suggestion
Any changes made to shallow clones affect only the clones themselves and not the
```